website/src/components/Search.js

```diff
@@ -4,15 +4,42 @@ import { jsx } from '@emotion/core';
 import { useEffect, useRef } from 'react';
 import { navigate } from 'gatsby';
 import { Input } from '@arch-ui/input';
+import { borderRadius, colors } from '@arch-ui/theme';
 
 import { addCallback } from '../utils/async-load-search';
-import { algoliaStyles } from '../utils/algolia-styles';
 
 const searchId = 'algolia-doc-search';
 
+function allowKeyboardRequest(event) {
+  if (event.key !== '/') {
+    return false;
+  }
+  if (['INPUT', 'TEXTAREA'].includes(document.activeElement.nodeName)) {
```

Comment:
Maybe this could just be

```
if (event.key !== '/' || ['INPUT', 'TEXTAREA'].includes(document.activeElement.nodeName)) {
  return false;
}
```

Reply:
It definitely could be, but I think this is clearer